src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -158,9 +158,19 @@ export const WorkspaceData = (signal) => ({
     const res = await fetchWDS(root)('status', _.merge(authOpts(), { signal }));
     return res.json();
   },
-  listInstances: async (root: string): Promise<any> => {
-    const res = await fetchWDS(root)('instances/v0.2', _.merge(authOpts(), { signal }));
-    return res.json();
+  listInstances: async (root: string, workspaceId: string): Promise<any> => {
+    try {
+      const response = await fetchWDS(root)(`collections/v1/${workspaceId}`, _.merge(authOpts(), { signal }));
+      const data = await response.json();
```

Comment:
`collections` and `instances` APIs return different response types (objects with workspaceId under `key` vs. lists of workspaceIds), so I massage the new collection responses to match what is already expected elsewhere.